### PR TITLE
Judge viz

### PR DESCRIPTION
**Pull request by paulstgermain**
## Description

This pull requests fixes a bug on the judgePage component which caused judge data to be unable to be pulled from DB to be displayed on the page due to changes made during the team merge. Also adds the visualization from the DS team by making the final link between the front-end and the back-end to send and receive the appropriate information.

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by nwestallen — APPROVED** [review]
Everything looks clean to me

File: src/components/pages/DataHub/DataHub.js [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+        headers: {
+          Authorization:
+            'Bearer ' +
+            'eyJraWQiOiJrMjJwZmRidEJsZlk1cGE0U1BMa1luWHp0SkVVY2MzS2hUbHJtZlRKNWxBIiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiIwMHVsemRyaXpFMnl6eFRvSDVkNiIsIm5hbWUiOiJUZXN0IEFkbWluIDAwMSIsImVtYWlsIjoibGxhbWEwMDFAbWFpbGRyb3AuY2MiLCJ2ZXIiOjEsImlzcyI6Imh0dHBzOi8vZGV2LTQxMzI3Nzg2Lm9rdGEuY29tL29hdXRoMi9kZWZhdWx0IiwiYXVkIjoiMG9hYmJxbW92SG5RVG9SQlA1ZDYiLCJpYXQiOjE2MjE2MDk3NTcsImV4cCI6MTYyMTYxMzM1NywianRpIjoiSUQuS1Npa2lJLWxueFlmNTNXZzA0aFFrNkNDVTgyU3NHWVItd1ptY3FwV2pIWSIsImFtciI6WyJwd2QiXSwiaWRwIjoiMDBvYmJzY2wyYlM3YzdjRkQ1ZDYiLCJub25jZSI6ImJwSFQ1TGg0Tmp0YlFWWkZzOWZScXg2WE9wTGlBVWhSZWxLdnpZcVBTTHdDUWQ3VXRCZkxHOUVjakJOZVM5cWEiLCJwcmVmZXJyZWRfdXNlcm5hbWUiOiJsbGFtYTAwMUBtYWlsZHJvcC5jYyIsImF1dGhfdGltZSI6MTYyMTYwOTc1MiwiYXRfaGFzaCI6IjNpazdMRWwyZjFDRVJoek5fb0NodVEifQ.Wn7ZQLCXCdvpehBKnTHwmCiWqNLg_X6WL0dr2RyaFNSKGVk_3u08vIKVHD0bPQVidgEb-G8PZ0kMH5gA4aaLotMdk6Ml4RV8B_z4l-1GYIWA2G5eJNg4c3C2CQOB3KobwNeg2PZArhXjPibFJwz0A6nOvwiWBz5A9C7GMv5ulgYVMlZRuOAB4UQLhnBpEwruJ3H6G06ISzJ7492kBlYe0jTDf8KwUQ9FMtHoSD5XexN6Mf6gUPOEjlOM2_MTf9N7jJHEVXgli2uI2HDAPV2WPwCdhQSKS2XIUNlc5sBu1jBVkMDWl8RA3suJnQsQvdcGOJFa3VCSPoxmuIELor89Pg',
```
Comment (WeiWu2) [inline src/components/pages/DataHub/DataHub.js:18]: Uh... correct me if I'm wrong but I believe this is not suppose to be here

Comment (JonathanMary) [reply src/components/pages/DataHub/DataHub.js:18]: You're right.

Comment (sebastian-espeset) [reply src/components/pages/DataHub/DataHub.js:18]: Good catch

**Review by katieolson84 — CHANGES_REQUESTED** [review]
There is a previous comment that should be resolved and then we can approve.

**Review by JonathanMary — APPROVED** [review]
Good to go, thanks for editing.

**Review by JonathanMary — APPROVED** [review]

**Review by JonathanMary — APPROVED** [review]

**Review by katieolson84 — APPROVED** [review]
Approved as the changes requested have been made. Great work!

**Pull request merged**